core/src/main/scala/org/apache/spark/storage/PushBasedFetchHelper.scala

```diff
@@ -142,7 +142,7 @@ private class PushBasedFetchHelper(
     val mergedBlocksMetaListener = new MergedBlocksMetaListener {
       override def onSuccess(shuffleId: Int, shuffleMergeId: Int, reduceId: Int,
           meta: MergedBlockMeta): Unit = {
-        logInfo(s"Received the meta of push-merged block for ($shuffleId, $shuffleMergeId," +
+        logDebug(s"Received the meta of push-merged block for ($shuffleId, $shuffleMergeId," +
```

Comment:
This change is unrelated to this PR. But given we found that this line could pollute logs in OSS version, and since this is just one single log level change, thus wanted to add it alone with this PR.